sdk/storage/Azure.Storage.DataMovement/tests/PauseResumeTransferTests.cs

```diff
@@ -842,8 +843,8 @@ public async Task TryPauseTransferAsync_DataTransfer_Directory(TransferDirection
             using DisposingLocalDirectory checkpointerDirectory = DisposingLocalDirectory.GetTestDirectory();
             using DisposingLocalDirectory sourceDirectory = DisposingLocalDirectory.GetTestDirectory();
             using DisposingLocalDirectory destinationDirectory = DisposingLocalDirectory.GetTestDirectory();
-            await using DisposingContainer sourceContainer = await GetTestContainerAsync();
-            await using DisposingContainer destinationContainer = await GetTestContainerAsync();
+            await using DisposingContainer sourceContainer = await GetTestContainerAsync(publicAccessType: PublicAccessType.BlobContainer);
+            await using DisposingContainer destinationContainer = await GetTestContainerAsync(publicAccessType: PublicAccessType.BlobContainer);
```

Comment:
Usually destination containers should not need to be public? Same below.

Reply:
Removed for the source, left for the destination.